functional-tests/functional/input/lime/VisibilityPlatform.lime

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2016-2022 HERE Europe B.V.
```

Comment:
Could you add checks when it's used with @Lang(EnableIf="...") ?

Reply:
Done